strapi/strapi
# feat(content-manager): add syntax highlighting in the code block type in the Blocks field

### What does it do?

Adds the highlight colors in the Code blocks inside Block Editor.

### Why is it needed?

We want to highlight the syntax of every single language.
We used Prismjs to do it and this theme https://prismjs.com/index.html?theme=prism-solarizedlight if we want something else just add a comment.

### How to test it?

 User adds a code block in the Blocks field
 - GIVEN THAT I have selected the code block type,
 - WHEN I enter code into the code block
   - THEN the code is displayed with syntax highlighting according to the language detected or specified

 User selects a specific programming language for syntax highlighting
 - GIVEN THAT I am editing a code block in the Blocks field,
 - WHEN I choose a programming language from a dropdown list (e.g., JavaScript, Python, HTML),
   - THEN the code block applies syntax highlighting specific to the selected language


### Related issue(s)/PR(s)

CS-865
